packages/docs/sidebars.ts

```diff
@@ -61,6 +61,12 @@ const sidebars: SidebarsConfig = {
       link: { type: 'doc', id: 'administration/provider-directory/index' },
       items: [{ type: 'autogenerated', dirName: 'administration' }],
     },
+    {
```

Comment:
As above, put this under care navigation

Reply:
updated link to be under careplan